presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -811,6 +811,15 @@ Optimizer Properties
 
     Track history based plan statistics from complete plan fragments in failed queries.
 
+``optimizer.exploit-constraints``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``true``
+
+    Enable analysis and propagation of logical properties (distinct keys, cardinality, etc.) among the nodes of
```

Comment:
```suggestion
    Enable analysis and propagation of logical properties like distinct keys or cardinality among the nodes of
```
Avoid Latin expressions or abbreviations in documentation. For context and suggestions, see [the Gitlab documentation style guide recommended word list entry for etc.](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#etc). 

I'm suggesting a possible revision, but if this one doesn't work for you please read the linked entry and propose an alternate revision.